kmp/reporters.go

```diff
@@ -111,12 +111,24 @@ func (r *ShortDiffReporter) Report(rs cmp.Result) {
 	t := cur.Type()
 	var diff string
 	// Prefix struct values with the types to add clarity in output
-	if !vx.IsValid() || !vy.IsValid() {
-		r.err = fmt.Errorf("Unable to diff %+v and %+v on path %#v", vx, vy, r.path)
+	if !vx.IsValid() && !vy.IsValid() {
+		r.err = fmt.Errorf("Both are invalid on path %#v", r.path)
```

Comment:
I would keep the original error message just for clarity.

Reply:
done